mmv1/third_party/terraform/services/netapp/resource_netapp_volume_test.go

```diff
@@ -141,6 +141,7 @@ resource "google_netapp_storage_pool" "default" {
     service_level = "PREMIUM"
     capacity_gib = "2048"
     network = data.google_compute_network.default.id
+    allow_auto_tiering = true
```

Comment:
this configuration update appears to be causing issues where it is unable to update since the Pool still has volumes:

```hcl
        Error: Error waiting for Deleting storagePool: Error code 3, message: invalid request error: "Error deleting pool - Pool still contains volumes".
```

Reply:
This error still looks to be present with the recent commits pushed.